Added style to FAQ page

Akshay-21 opened the pull request:
Hey Hi @robogyan-adgitm i Added style for FAQ page. I have created PR kindly check it and merge.
Thank You!

Akshay-21 committed:
Added style to FAQ page

Akshay-21 committed:
resolve conflict of FAQ page style

Akshay-21 committed:
resolved conflict of FAQ page style

Akshay-21 committed:
resolved conflict style of FAQ page

robogyan-adgitm commented on dist/css/main.scss:
```diff
@@ -350,36 +368,38 @@ footer{
+    .footer-container {
+        &-links {
+            li {
+                a {
```
All the changes in this file are spam

Akshay-21 replied:
Ok i will check this issue.

Akshay-21 replied:
Removed these unused things.
kindly resolve conversation.
Thank You!

robogyan-adgitm commented on faq.html:
```diff
@@ -54,68 +55,438 @@ <h1>Robogyan</h1>
   </main>
+
+  <!-- Akshay Kumar Maurya -->
+  <!-- ================================================================== -->
```
No personal info in code allowed

Akshay-21 replied:
This one also i will check and remove this.

Akshay-21 replied:
Removed Personal info.
kindly resolve conversation.
Thank You!

Akshay-21 commented:
Hey Hi @robogyan-adgitm after this comment i will see this and what do i need to do for this issue.

robogyan-adgitm commented:
Just resolve these two things and recreate the PR

Akshay-21 committed:
remove name info

Akshay-21 committed:
removed unused file and personal info, i have updated FAQ Accordion.

Akshay-21 commented:
> Just resolve these two things and recreate the PR

Hi @robogyan-adgitm, i have removed unused file and code and personal info, i have updated FAQ Accordion using smooth design with theme matching.

Please kindly check this.

Thank You!

![image](https://user-images.githubusercontent.com/32409386/194649756-0ca74e6c-3d6f-4d3f-8bfb-17ae3e93e1b4.png)

Akshay-21 committed:
Resolved new conflict

robogyan-adgitm reviewed (approved):
Good

robogyan-adgitm commented:
Not be merging this as from the design it's not clear that the unexpanded questions could be expanded. And for now current UI for the FAQs is better. Marked as hacktoberfest-accepted though as it is a good enough contribution to be worked on later.

Akshay-21 commented:
> Not be merging this as from the design it's not clear that the unexpanded questions could be expanded. And for now current UI for the FAQs is better. Marked as hacktoberfest-accepted though as it is a good enough contribution to be worked on later.

Thank You! for accepting the PR.
